scrapegraphai/nodes/rag_node.py

```diff
@@ -92,6 +92,8 @@ def execute(self, state):
         if isinstance(embedding_model, OpenAI):
             embeddings = OpenAIEmbeddings(
                 api_key=embedding_model.openai_api_key)
+        elif isinstance(embedding_model, AzureOpenAIEmbeddings):
```

Comment:
DId you have the chanche to test with azure embedding model? Did you use text-ada? does it work properliy? :)

Reply:
yes, I used text-ada, have attached example as well.